ensime_shared/config.py

```diff
@@ -5,14 +5,16 @@
 BOOTSTRAPS_ROOT = os.path.join(os.environ['HOME'], '.config/ensime-vim/')
 """Default directory where ENSIME server bootstrap projects will be created."""
 
+LOG_FORMAT = '%(levelname)-8s <%(asctime)s> (%(filename)s:%(lineno)d) - %(message)s'
+
 gconfig = {
     "ensime_server": "ws://127.0.0.1:{}/jerky",
     "localhost": "http://127.0.0.1:{}/{}",
 }
 
 feedback = {
     "analyzer_ready": "Analyzer is ready",
-    "displayed_type": "The type {} has been displayed",
+    "displayed_type": "The type %s has been displayed",
```

Comment:
Unrelated, but why are this kind of messages defined here if they are not feedback to the user? They might be moved to the log statement itself.